app/assets/stylesheets/mtl/extend/_avatars.scss

```diff
@@ -1,24 +1,26 @@
 // avatars
 .avatar,
 .mtl-layout-default .side-nav .avatar {
-  display: block;
-  width: 52px;
-  height: 52px;
-  line-height: 52px;
-  padding: 0;
-  border-radius: 26px;
+  display: inline-block;
+  border-radius: 50%;
+  height: 50px;
```

Comment:
so avatar-small, avatar-medium and avatar-large, do these correspond to the icon sizes in some way?

```html
      <!--
      Sizes:
      tiny: 1rem
      small: 2rem
      medium: 4rem
      large: 6rem
      -->
      <i class="large material-icons">insert_chart</i>
```
http://materializecss.com/icons.html